types/query.d.ts

```diff
@@ -136,40 +173,11 @@ declare module 'mongoose' {
      * Another alias for the `new` option. `returnOriginal` is deprecated so this should be used.
      */
     returnDocument?: 'before' | 'after';
-    /**
-     * Set to true to enable `update validators`
-     * (https://mongoosejs.com/docs/validation.html#update-validators). Defaults to false.
-     */
-    runValidators?: boolean;
-    /* Set to `true` to automatically sanitize potentially unsafe user-generated query projections */
-    sanitizeProjection?: boolean;
```

Comment:
this options was removed without being added to `MongooseSpecificQueryOptions`, is this intended?